homeassistant/components/squeezebox/media_player.py

```diff
@@ -129,9 +160,7 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     hass.data.setdefault(DOMAIN, {})
     hass.data[DOMAIN].setdefault(config_entry.entry_id, {})
 
-    known_players = hass.data[DOMAIN].get(KNOWN_PLAYERS)
-    if known_players is None:
-        hass.data[DOMAIN][KNOWN_PLAYERS] = known_players = []
+    known_players = hass.data[DOMAIN].setdefault(KNOWN_PLAYERS, [])
```

Comment:
Instead of storing the entities we should just store a dict that maps player id to entity_id.

Reply:
Hmm, the user can change the entity_id, so a static map doesn't work.

Reply:
Let's keep this constainer with entities for now.